automatic/jenkins/jenkins.nuspec

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Do not remove this test for UTF-8: if appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+    <metadata>
+        <id>jenkins</id>
+        <version>2.222.4</version>
+        <packageSourceUrl>https://github.com/chocolatey-community/chocolatey-packages/tree/master/automatic/jenkins</packageSourceUrl>
+        <owners>chocolatey-community,pauby,johnypony3</owners>
+        <title>Jenkins CI</title>
+        <authors>Kohsuke Kawaguchi</authors>
+        <projectUrl>https://www.jenkins.io/</projectUrl>
+        <iconUrl>https://cdn.jsdelivr.net/gh/git@github.com:chocolatey-community/chocolatey-packages/icons/jenkins.png</iconUrl>
+        <copyright>2018 Kohsuke Kawaguchi</copyright>
```

Comment:
Is this the latest copyright available?

Reply:
Could potentially have been part of the website itself (doesn't look like it is now though).

Let us use the one from the License in the GitHub repository.